recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - jigsaw >=0.9.13
+    - jigsaw >=0.9.13.1
```

Comment:
@dengwirda, is this correct or is each jugsawpy version **only** for that jigsaw version and I should remove `>=`?

Reply:
@xylar, officially yes --- I try to keep each `jigsawpy` version in-sync with the underlying `jigsaw` (since they share a few data structures, etc), so the `jigsawpy-0.3.2` release includes the `jigsaw-0.9.13.1` src as a sub-tree.

Can I check what the `- jigsaw >= 0.9.13.1` line actually does here though --- is this referencing the `jigsaw` GitHub release, or the conda package?

Reply:
It refers to the conda package. With the current constraints, I could (for example) keep using jigsawpy 3.1 with the latest jigsaw.

Reply:
Yes, I think that's right --- since the MPAS workflows build `jigsaw` on it's own, both `jigsawpy-0.3.2` and `jigsawpy-0.3.1` should work just as well (as there's no change to the `python` layer involved).

A lot of non-E3SM `jigsawpy` users compile `jigsaw` direct from the sub-tree though, so I'm keeping everything in-sync for this.

Reply:
Oh, okay.  I just switched to pinning an exact version.  I can switch it back and do a new build if you think that's better.

It would be good to have an upper bound if you know you plan to break backwards compatibility at some point.  For example:
```
jigsaw >=0.9.13.1,<0.10.0
```
I don't know if you expect backwards compatibility for the major, minor or micro version numbers.  That's something you have to decide.

Reply:
> since the MPAS workflows build `jigsaw` on it's own

Just to be 100% clear, MPAS just uses the `jigsaw` conda package (which contains compiled libraries and executables) so doesn't build the source.

Reply:
Generally, I try never to break backward compatibility, so I guess there should be a fair amount of safety when using an older `jigsawpy` with a newer `jigsaw`. 

Due to the sub-tree business though, I expect to always make a new `jigsawpy` release for each `jigsaw` release, so hopefully pinning them to be exactly in-sync will continue to make sense.